library/src/main/java/com/pixplicity/htmlcompat/HtmlToSpannedConverter.java

```diff
@@ -292,7 +301,9 @@ Spanned convert() {
     }
 
     private void handleStartTag(String tag, Attributes attributes) {
-        if (tag.equalsIgnoreCase("br")) {
+        if (mTagHandler != null && mTagOverrides.contains(tag)) {
```

Comment:
before default handling kicks in, we check if the tag is one that we want to override handling for.
`mTagOverrides` is created from varargs